cygnustechnologies/pocket-goblin-client
# Server writes to database after GET to Plaid's /transactions endpoint

To keep things easier for everybody, I chain promises to create the entire structure and populate the database from that call. 

Simplified the dbsync.js file. Note that if you want to test multiple times, uncomment the line that has {force:true} in order to destroy the db before re-creating. Otherwise, you will get a lot of duplicate data each time you hit the /transactions endpoint. 

After testing tomorrow, we can improve the way this works. I didn't want to do too much until syncing with JP. 

Note: It seems that we can drastically simplify the db structure, but we'll hold off on that until we are fully done with the MVP

Once merged, to test this out follow this work flow: 
- create 'pocketgoblin' database in your local postgres if you did not already create
- node dbsync.js                 Use this every time you want to clear out the database (ensure {force:true} is set) 
- npm run server-dev
- using postman, or similar, attempt a get at localhost:8000/transactions  in order to kick off the call to plaid and the saves to the database. 
- check the database to see all created data.
